src/settings.py

```diff
@@ -19,8 +19,8 @@ class Settings(BaseSettings):
     REDIS_USERNAME: str = os.getenv("REDIS_USERNAME")
     REDIS_PASSWORD: str = os.getenv("REDIS_PASSWORD")
     OPENAI_API_KEY: str = os.getenv("OPENAI_API_KEY")
-    BACKEND_CORS_ORIGINS: list[AnyHttpUrl] = []
-    GENERATION_JOB_PREFFIX_KEY: str = "GJOB"
+    BACKEND_CORS_ORIGINS: list[AnyHttpUrl] = os.getenv("ORIGINS").split(",")
```

Comment:
We should update our `env_template`

Reply:
You might notice that the code on L22 does't contain the split anymore. 

This is because I have defined a validator function that handles the validation and splitting of the string below.